# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lame/hotfix.patch
# Copyright (C) 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- lame-3.99.5/libmp3lame/vector/xmm_quantize_sub.c.vanilla	2014-11-30 14:41:48.644000000 +0000
+++ lame-3.99.5/libmp3lame/vector/xmm_quantize_sub.c	2014-11-30 14:42:08.408000000 +0000
@@ -34,6 +34,8 @@
 
 #ifdef HAVE_XMMINTRIN_H
 
+#pragma GCC target ("sse2")
+
 #include <xmmintrin.h>
 
 typedef union {
